apps/platform-app/src/app/[locale]/(profile)/(components)/VerificationModal/VerificationModal.tsx

```diff
@@ -0,0 +1,57 @@
+'use client'
+
+import { getProfileByUserId } from '@/backend/profile/profile.service'
+import { Button } from '@gdh/ui-system'
+import { PublishingState } from '@prisma/client'
+import styles from './VerificationModal.module.scss'
+import { GithubIcon } from '@gdh/ui-system/icons'
+
+interface VerificationModalProps {
+  profileStatus: PublishingState;
+  onClose: () => {}
+}
+
+export default async function VerificationModal({
+  profileStatus,
+  onClose
+}: VerificationModalProps) {
+  //the text for the modal isn't added to translations yet
+  // const t = useTranslations(I18nNamespaces.Buttons)
+  // const router = useRouter()
+
+  // const handleEditClick = () => {
+  //   router.push(AppRoutes.editProfile)
+  // }
+
+  //quick exit from the function if nothing to display
+  if (profileStatus === PublishingState.DRAFT || profileStatus === PublishingState.PENDING)
```

Comment:
**issue**: The condition should check if `profileStatus` is `PublishingState.APPROVED` instead of using `PublishingState.APPROVED` directly in the ternary operation.